example/Mock.elm

```diff
@@ -26,22 +22,6 @@ add =
     (+)
 
 
-{-|
-
-    inc $ inc 1 --> 3
-
-    add : Int -> Int -> Int
-    add = (+)
-
-    inc : Int -> Int
-    inc x = add 1 x
-
--}
-($) : (a -> b) -> a -> b
```

Comment:
I'm fine with dropping support for infix-operators

Reply:
Seems okay